Solution1/BusinessLayer/DTOs/ForecastWeatherDTO.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusinessLayer.DTOs
+{
+    public class ForecastWeatherDTO
+    {
+        public List<WeatherForDate> WeatherValuesForPeriod { get; set; }
+
+        public string CityName { get; set; }
+    }
+
+    public class WeatherForDate
```

Comment:
add class BaseWeatherDTO, inherit WeatherDTO, and this as well

Reply:
+